integration-test-groups/xml/native/classpath/src/main/java/org/apache/camel/quarkus/component/xml/it/FunctionsConfiguration.java

```diff
@@ -18,6 +18,10 @@
 
 import io.quarkus.runtime.annotations.RegisterForReflection;
 
-@RegisterForReflection(targets = { org.apache.camel.quarkus.test.support.xslt.Functions.class })
+@RegisterForReflection(targets = {
+        org.apache.camel.quarkus.test.support.xslt.Functions.class,
+        org.apache.camel.quarkus.test.support.xslt.MyExtensionFunction1.class,
+        org.apache.camel.quarkus.test.support.xslt.MyExtensionFunction2.class
+})
```

Comment:
Do we doc for this ? Maybe it's https://camel.apache.org/camel-quarkus/3.0.x/reference/extensions/xslt.html#extensions-xslt-configuration-extension-functions-support or similar.

Reply:
Sorry, I think it can not be supported in `native` mode right now. So it should be removed.